docs/2.5. APIs - Query Parameters.md

```diff
@@ -333,7 +343,27 @@ GET /x-nmos/query/v1.0/flows?tags.location=Salford&tags.location=London
 
 ## Advanced (RQL) Queries (Optional)
 
-Query APIs MAY support Resource Query Language (RQL) queries against their API resources where the 'rql' trait is specified in the RAML documentation. A 501 HTTP status code MUST be returned where a RAML query is attempted using RQL functions or operators which are not supported by a Query API.
+Query APIs MAY support Resource Query Language (RQL https://github.com/persvr/rql/) queries against their API resources where the 'rql' trait is specified in the RAML documentation. A 501 HTTP status code MUST be returned where a RAML query is attempted using RQL functions or operators which are not supported by a Query API.
+
+RQL should be formatted in the normalised form as opposed to using FIQL syntax, and passed via the query string using '?query.rql=...'.
+
+Querying within objects and arrays should be performed as in the Basic Queries case by using the '.' separator.
+
+When an RQL query is specified, the Basic Queries format MAY be ignored in order to simplify implementation.
+
+Note that as RQL permits the definition of very complex queries, the server may return a 400 error code to indicate that it is refusing to action the client's request due to the query's complexity.
+
+### Constraints
+
+Not all RQL operators will be suitable for use with a Query API. It is suggested that the following operators are supported, however these are not mandated and implementations may choose to support their own subset.
+
+1) Functions: in(), out(), sort(), like(), select()
```

Comment:
Remove 'sort()'?

Reply:
Where does 'like()' come from? Don't see it at https://github.com/persvr/rql/search?q=like&unscoped_q=like.

Reply:
I suspect this was from another reference when written. The intent was to have something which allowed a regex style. I'll remove it for now in the absence of 'matches()' in the formal docs. In time these should be replaced by explicit instructions for how they should be implemented.